docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,141 @@
+= AKS Recover cluster objects from Azure Blob Storage backup
+
+== Prerequisites
+
+* Executables used in this guide:
+** `kubectl`
+** `yq`
```

Comment:
Specify the required yq version. yq 3.x and 4.x have completely different invocations.

Reply:
Still needs to be addressed.

Reply:
The one we use at the cloudscale and exoscale cluster installation guide is this one in Go: https://github.com/mikefarah/yq